doc/https.md

```diff
@@ -0,0 +1,70 @@
+# HTTPS
+
+Some web technologies require a secure context when not operating over `localhost`. This includes things link clipboard access and WebRTC--two technologies used within the `FractalSys` applet. For this reason, we need a means of running psinode with an SSL.
+
+## Generating certs on local machine
+
+1. Install `mkcert` locally
```

Comment:
What's "local," here? Is it within the container? Or on your host machine and then you copy them into the docker container if that's what you're using?

Reply:
host machine. I'll clarify.